components/molecule/textareaField/src/index.js

```diff
@@ -18,22 +18,26 @@ const getState = ({successText, errorState, alertText}) => {
   if (alertText) return AtomTextareaStates.ALERT
 }
 
+const NOOP = () => {}
+
 const MoleculeTextareaField = ({
+  alertText,
+  autoHideHelpText = false,
+  errorText,
+  exceedLength = false,
```

Comment:
```canExceedLength``` or similar maybe could be more descriptive. NBC

Reply:
Great naming, I'm so bad at it! Thks!